test/trento/application/projectors/database_projector_test.exs

```diff
@@ -295,6 +297,67 @@ defmodule Trento.DatabaseProjectorTest do
                      1000
   end
 
+  test "should broadcast database_instance_absent_at_changed when DatabaseInstanceMarkedAbsent event is received" do
```

Comment:
The main things under test is the projection, not the broadcasting.
Let's change the test text to:
`should update the absent_at field on DatabaseInstanceMarkedAbsent event is received`.

The same for others